recipes/boost/all/test_package/conanfile.py

```diff
@@ -56,7 +54,6 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not can_run(self):
-            return
-        with chdir(self, self.folders.build_folder):
-            self.run(f"ctest --output-on-failure -C {self.settings.build_type}", env="conanrun")
+        if can_run(self):
+            cmake = CMake(self)
+            cmake.test()
```

Comment:
bear in mind that one of the reasons we used `ctest` rather than `cmake.test()` is to ensure that in the event of  the `tools.build:skip_test` config being abled, this test still runs unconditionally, as `cmake.test()` generally applies to library tests rather than executables in the test package